tests/test_deepgrow_dataset.py

```diff
@@ -10,47 +10,93 @@
 # limitations under the License.
 
 import os
+import shutil
 import tempfile
 import unittest
 
 import nibabel as nib
 import numpy as np
+from parameterized import parameterized
 
 from monai.apps.deepgrow.dataset import create_dataset
 
+TEST_CASE_1 = [{"dimension": 2, "pixdim": (1, 1)}, {"length": 3}, 9, 1]
+
+TEST_CASE_2 = [{"dimension": 2, "pixdim": (1, 1), "limit": 1}, {"length": 3}, 3, 1]
+
+TEST_CASE_3 = [{"dimension": 2, "pixdim": (1, 1)}, {"length": 1}, 3, 1]
+
+TEST_CASE_4 = [{"dimension": 3, "pixdim": (1, 1, 1)}, {"length": 1}, 1, 1]
+
+TEST_CASE_5 = [{"dimension": 3, "pixdim": (1, 1, 1)}, {"length": 1, "image_channel": 4}, 1, 1]
+
+TEST_CASE_6 = [{"dimension": 2, "pixdim": (1, 1)}, {"length": 1, "image_channel": 4}, 3, 1]
+
+TEST_CASE_7 = [
+    {"dimension": 2, "pixdim": (1, 1), "label_key": None},
+    {"length": 1, "image_channel": 4, "with_label": False},
+    40,
+    None,
+]
+
+TEST_CASE_8 = [
+    {"dimension": 3, "pixdim": (1, 1, 1), "label_key": None},
+    {"length": 1, "image_channel": 4, "with_label": False},
+    1,
+    None,
+]
+
 
 class TestCreateDataset(unittest.TestCase):
-    def _create_data(self, tempdir):
+    def setUp(self):
```

Comment:
could you also add set_determinism to setUp/tearDown, e.g. https://github.com/Project-MONAI/MONAI/blob/db831ae096bec76371bb8c2e3db226d947a5ac06/tests/test_bounding_rect.py#L28-L32 because np.random is used in the test cases

Reply:
done